### PR TITLE
PD-HTML/CSS

### PR DESCRIPTION
**Pull request by esma-g**
Updated PD content and coursework

## What does this change?

Module: HTML/CSS
Week(s): 1, 2, 3 ,4

## Description

Updated PD content and coursework

### CODE REVIEW

File: docs/html-css/week-1/homework.md [hunk 1]
```diff
@@ -46,3 +46,31 @@ Complete the [Flexbox Froggy game](https://flexboxfroggy.com) to practise using
...
+### Create a study schedule (PD - 1.5 hr)
+
+Over the next months you will have lots of coursework to complete! How will you manage it all? 
+
+Create a study schedule. 
+
+This activity will give you practice using Google sheets to create and maintain a schedule. 
+
+Go to the link below and do the course. Completed the quiz at the end of the course. We will be able to see your responses.
+[Create a goal setting worksheet] (https://applieddigitalskills.withgoogle.com/c/middle-and-high-school/en/create-a-study-schedule-to-meet-your-goals/overview.html)
```
Comment (kfklein15) [inline docs/html-css/week-1/homework.md]: I think there is a space between ] and (, which means the link might not show correctly. But I could be wrong.

**Review by kfklein15 — DISMISSED** [review]
Reviewed. I am unsure about one of the URL links, but let's get this merged and see how it looks.

File: docs/html-css/week-1/homework.md (hunk 1 above)
Comment (esma-g) [reply docs/html-css/week-1/homework.md]: You are correct, that space shouldn't have been there. I'm trying to fix it now.

**Review by kfklein15 — APPROVED** [review]
All good.

**Comment by kfklein15** [comment]
@esma-g , I merged it into production. I can see the PD changes to the lessons but not on the coursework. Can you check if it is all ok, please?

**Comment by esma-g** [comment]
> @esma-g , I merged it into production. I can see the PD changes to the lessons but not on the coursework. Can you check if it is all ok, please?

It look all good, thanks.

**Pull request merged**